lib/index.d.ts

```diff
@@ -6,7 +6,7 @@ export type CreateOptions = {
   globOptions?: GlobOptions;
   ordering?: string;
   pattern?: string;
-  transform?: (filePath: string) => string;
+  transform?: (filePath: string) => NodeJS.ReadWriteStream | void;
```

Comment:
I don't think `void` can work in `stream.pipeline`?

Reply:
@malept If transform has no truthy return value, pipeline won't be called.

Reply:
Oh right. But I don't think we want to encourage that, I think that's mostly a check for JavaScript since there's no types.

Reply:
@malept and i see readme

> You can pass in a transform option, that is a function, which either returns nothing, or a stream.Transform. The latter will be used on files that will be in the .asar file to transform them (e.g. compress).

so i think `void` is good.

Reply:
I think `void` needs to be deprecated and then removed in the next major release.

Reply:
i see. i have changed it.

Reply:
I thought about it some more and it makes a certain amount of sense that `void` is a valid return value. I've updated the definition & the test accordingly.